HIRS_Utils/src/main/java/hirs/data/persist/IMAPolicy.java

```diff
@@ -457,9 +457,11 @@ public final void setPartialPathEnable(final boolean partialPathEnable) {
         Multimap<String, String> equivalentPaths = HashMultimap.create();
 
         // define equivalences
-        equivalentPaths.put("/bin/", "/usr/bin/");
-        equivalentPaths.put("/lib/", "/usr/lib/");
-        equivalentPaths.put("/lib64/", "/usr/lib64/");
+        equivalentPaths.put("/bin/",     "/usr/bin/");
+        equivalentPaths.put("/lib/",     "/usr/lib/");
+        equivalentPaths.put("/lib64/",   "/usr/lib64/");
+        equivalentPaths.put("/usr/bin/", "/usr/sbin/");
+        equivalentPaths.put("/sbin/",    "/usr/sbin/");
 
```

Comment:
Does one still need to do equivalent paths if check is done by hash? Is there some benefit to leaving this in such as number of db accesses?

Reply:
I think these are still needed for the case where path and hash both need to "match"--meaning equivalent. However, there are some transitive equivalencies that are not captured here. For example, `/bin` is equivalent to `/usr/bin` which is equivalent to `/usr/sbin` which is equivalent to `/sbin`. Perhaps a list of equivalent sets would be easier to manage.

Reply:
Yes, these are still needed because they still inform matches & mismatches.  Checking by hash is only done for records that are otherwise unknown.

Although the term 'equivalent' would suggest these paths should be transitive, we've only been adding explicit equivalent pairs so far, and not making these transitive to more tightly capture the actual installation locations we're seeing in RPMs.